evm.md

```diff
@@ -339,6 +340,16 @@ The `#next` operator executes a single step by:
          <pc> PCOUNT </pc>
          <program> ... PCOUNT |-> OP ... </program>
       requires EXECMODE in (SetItem(NORMAL) SetItem(VMTESTS))
+
+      rule <mode> COVERAGE </mode>
+           <k> #next 
+            => #setMode NORMAL
+            ~> #next
+            ~> #setMode COVERAGE
+            ...
+           </k> 
+           <pc> PCOUNT </pc>
+           <analysis> ... "coverage" |-> (PCS:Set (.Set => SetItem(PCOUNT))) ... </analysis>
```

Comment:
We should be able to put this rule in `analysis.md` instead of here as well.

Reply:
@ana-pantilie i think you can add the following rule, and keep the default configuration of `<analysis> .Map </analysis>` in the above.

```
rule
<mode> COVERAGE </mode>
<k> #next </k>
<analysis> M => M [ "coverage" <- .Set ] </analysis>
requires "coverage" not in_keys(M)
```